On-prem: Initial docs for embedded cluster install (#288)
SpecterOps/bloodhound-docs: docs/on-premises/overview.mdx

---

> title: BloodHound Enterprise On-premises

Reviewer: I don't like the repeated use of "BloodHound Enterprise On-premises". That makes it sound like a different version of the product, when in reality it's a modifier on the purchased version. I would rather say something like "On-Premises BloodHound Enterprise" or "BloodHound Enterprise hosted within your own premises" or some other phraseology that doesn't feel like it would show up in a "Compare versions" table.

Author: Good points.

I'd like to try "Self-hosted BloodHound Enterprise". Even though that still wraps, I think it reads more naturally than "On-premises BloodHound Enterprise".

My only concern is that might not be aligned with our marketing material, which appears to use "BloodHound Enterprise On-premises".

sidebarTitle: Overview

> description: Learn about BloodHound Enterprise on-premises, a self-hosted deployment option that gives you full control over your infrastructure and data.

Reviewer: ```suggestion
description: Learn about self-hosted deployment options for BloodHound Enterprise, giving you full control over your infrastructure and data.
```

- **Network isolation** - Run in air-gapped or restricted network environments

## SaaS vs on-premises

Both deployment options provide the same core BloodHound Enterprise functionality, but differ in infrastructure management and control.

**Choose on-premises if you:**

- Require data to remain within your infrastructure
- Have regulatory or compliance requirements for data residency
- Need full control over the deployment environment
- Have existing infrastructure and operational expertise
- Prefer to manage updates and maintenance on your own schedule

**Choose SaaS if you:**

- Want SpecterOps to manage infrastructure and updates
- Don't have dedicated infrastructure or Kubernetes expertise
- Want automatic updates and new features as they're released
- Prefer predictable operational costs

## Deployment options

BloodHound Enterprise on-premises supports the following options for on-premises deployments:

| Option | Best fit | Key tradeoffs |
|--------|----------|--------------|
| **Deployment** | | |
| Embedded cluster | No existing Kubernetes | Bundled Kubernetes and application update together; less integration with existing K8s tooling |

> | KOTS | Mature Kubernetes operations | You manage the cluster lifecycle separately; higher operational complexity |

Reviewer: The typical acronym is "COTS," meaning "Consumer Off The Shelf." "KOTS" is not a common term and needs definition.

Author: I'll add descriptions for both deployment options before the table.

| **Installation** | | |
| Online | Internet access allowed | Lower install complexity; outbound dependencies required |

> | Air gap | No outbound internet access allowed | Higher install complexity; private registry required |

Reviewer: Agreed with the 🐇 here. This table is confusing as to what you're trying to communicate.
